aiida_quantumespresso/cli/calculations/epw.py

```diff
@@ -0,0 +1,175 @@
+
+    expected_process_type = 'aiida.calculations:quantumespresso.ph'
+    if ph_parent.process_type != expected_process_type:
+        raise click.BadParameter(
+            'The input calculation node has a process_type: {}; should be {}'.format(
+                ph_parent.process_type, expected_process_type
+            )
+        )
+
+    ph_parent_folder = ph_parent.get_outgoing(node_class=orm.RemoteData, link_label_filter='remote_folder').one().node
+
+    qibz_ar = []
+    for key, value in ph_parent.outputs.output_parameters.get_dict().items():
+        if key.startswith('dynamical_matrix_'):
+            qibz_ar.append(value['q_point'])
+
+    qibz_node = orm.ArrayData()
+    qibz_node.set_array('qibz', np.array(qibz_ar))
+
+    inputs = {
+        'code':
+        code,
+        'qpoints':
+        qpoints_mesh,
+        'kpoints':
+        kpoints_mesh,
+        'qfpoints':
+        qfpoints_mesh,
+        'kfpoints':
+        kfpoints_mesh,
+        'qibz':
+        qibz_node,
+        'parameters':
+        orm.Dict(
+            dict={
+                'INPUTEPW': {
+                    'nbndsub': 8,
+                    'elph': True,  # default is false
+                    #                    'kmaps': False, # default is false
+                    'epbwrite': True,  # default is false
+                    #                    'epbread': False, # default is false
+                    'epwwrite': True,  # default is false
+                    #                    'epwread': False, # default is false
+                    'proj(1)': 'Si : sp3',
+                    'elecselfen': True,
+                    #                    'nkf1': 6,
+                    #                    'nkf2': 6,
+                    #                    'nkf3': 6,
+                    #                    'nqf1': 2,
+                    #                    'nqf2': 2,
+                    #                    'nqf3': 2,
+                    'wannierize': True,
+                    'dvscf_dir': './save/',
+                    'dis_win_max': 18,
+                    'dis_froz_max': 8.5
+                }
+            }
+        ),
+        'parent_folder_nscf':
+        pw_nscf_parent_folder,
+        'parent_folder_ph':
+        ph_parent_folder,
+        'metadata': {
+            'options': get_default_options(max_num_machines, max_wallclock_seconds, with_mpi),
+        }
+    }
+
+    launch.launch_process(CalculationFactory('quantumespresso.epw'), daemon, **inputs)
+
+
+# Example to run an EPW calculation with AiiDA
+#
+# 1) Configure the pw.x, ph.x and epw.x codes called here pw@localhost, ph@localhost and epw@localhost
+#
+# 2) Run scf calculation
+# aiida-quantumespresso calculation launch pw -X pw@localhost -p   SSSP_1.1_efficiency
+# Retrieve the node PK (named here NODE_PK_SCF)
+#
+# 3) Run a phonon calculation
+# aiida-quantumespresso calculation launch ph -X ph@localhost  -C NODE_PK_SCF
+# Retrieve the node PK (named here NODE_PK_PH)
```

Comment:
This should still be moved to the documentation. You can add it as a section `Example run` in the page you already added